src/Microsoft.Azure.WebJobs.Host/Bindings/SystemBindingData.cs

```diff
@@ -36,14 +36,14 @@ internal class SystemBindingData
         public string MethodName { get; set; }
 
         /// <summary>
-        /// Get the current UTC date. 
+        /// Get the current UTC date. This returns the current value each time it is called.
         /// </summary>
-        public DateTime UtcNow => DateTime.UtcNow;
+        public DateTime DateTime => DateTime.UtcNow;
 
         /// <summary>
         /// Return a new random guid. This create a new guid each time it's called.
         /// </summary>
-        public Guid RandGuid => Guid.NewGuid();  
```

Comment:
We could leave it RandGuid. I just always thought "rand" was strange - it's a new guid so its randomness is implicit. Also there is a symmetry with "DateTime" if we just use "Guid" - it's the type name. I could go either way on this.